apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -38,15 +71,41 @@ class ItineraryCard extends React.Component<Props> {
       amount: parseFloat(data.price?.amount) ?? 0,
       currency: data.price?.currency ?? 'CZK',
     };
+    const localizedPrice = formatPrice(priceObject);
 
     return (
-      <ItineraryCardWrapper localizedPrice={formatPrice(priceObject)}>
-        <FlatList
-          data={data.sectors}
-          keyExtractor={this.keyExtractor}
-          renderItem={this.renderSectorItem}
-        />
-      </ItineraryCardWrapper>
+      <View
+        style={[
+          styles.container,
+          (hovered || detailOpen) && styles.containerElevated,
+        ]}
+      >
+        <Hoverable
```

Comment:
We didn't refactor this yet 🙈

Reply:
Sadly not. I'll try to find some time for it by the end of this week and then refactor this one.

Reply:
This task will probably go to `kiwi-academy`. I think we can wait for it.